ci/packages/package.go

```diff
@@ -140,11 +139,11 @@ func PackageWindowsAmd64() error {
 
 // PackageAndroid builds and stores Android package
 func PackageAndroid() error {
-	job.Precondition(func() bool {
-		pr, _ := env.IsPR()
-		fullBuild, _ := env.IsFullBuild()
-		return !pr || fullBuild
-	})
+	//job.Precondition(func() bool {
+	//	pr, _ := env.IsPR()
+	//	fullBuild, _ := env.IsFullBuild()
+	//	return !pr || fullBuild
+	//})
```

Comment:
Are we keeping it commented?

Reply:
@soffokl See commit message 👯

Reply:
The last commit was just for testing it on PR.

Reply:
I'm deleting that before merge.